prime_directive/core/scribe.py

```
@@ -42,9 +66,9 @@ async def generate_sitrep(
 ) -> str:
     """
     Generate a concise SITREP (situation report) summarizing repository state, recent terminal output, and optional human/task context.
-    
+
     Constructs a prompt from repo_id, git_state, terminal_logs, active_task, and human_* fields, then requests a short (2–3 sentence) SITREP with an immediate next step from the configured provider and model. By default uses Ollama; can use OpenAI as the primary provider or as a fallback. When OpenAI is used and db_path is provided, the function will check monthly budget and log estimated token usage and cost.
```

Ruff (0.15.4)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)